ddtrace/internal/module.py

```diff
@@ -494,6 +517,22 @@ def unregister_module_hook(cls, module, hook):
         except ValueError:
             raise ValueError("Hook %r not registered for module %r" % (hook, module))
 
+    @classmethod
+    def register_pre_exec_module_hook(cls, cond, hook):
```

Comment:
Would we ever have the need to unregister these hooks? Perhaps if IAST is disabled at some point?

Reply:
That would only really make sense if IAST is enabled and disabled before the module is executed, because if it is enabled and executed, its code would be AST-patched. AFAIK, the planned way to "disable" IAST is to do early returns in the IAST swapped functions